mithril-common/src/cardano_block_scanner/chain_reader_block_streamer.rs

```diff
@@ -239,11 +242,19 @@ mod tests {
                     Vec::<&str>::new(),
                 ),
             },
+            ChainBlockNextAction::RollForward {
+                parsed_block: ScannedBlock::new(
+                    "hash-3",
+                    BlockNumber(3),
+                    SlotNumber(30),
+                    Vec::<&str>::new(),
+                ),
+            },
         ])));
         let mut block_streamer = ChainReaderBlockStreamer::try_new(
-            chain_reader,
+            chain_reader.clone(),
             None,
-            BlockNumber(100),
+            BlockNumber(2),
```

Comment:
We no longer test the case where the block number is greater than the last block.
Is this case no longer relevant ?

Reply:
You're right, I added back the test :+1: